experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -77,6 +77,24 @@ class Flowables {
         std::forward<OnSubscribe>(function)));
   }
 
+  template <typename T>
+  static Reference<Flowable<T>> empty() {
+    auto lambda = [](Subscriber<T>& subscriber, int64_t) {
+      subscriber.onComplete();
+      return std::make_tuple(static_cast<int64_t>(0), true);
+    };
+    return Flowable<T>::create(std::move(lambda));
+  }
+
+  template <typename T>
+  static Reference<Flowable<T>> error(const std::exception_ptr ex) {
```

Comment:
Yes I agree, was wondering this when I wrote it but figured there was some reason the interface was changed to exception_ptr.